File: solutions/py/8 kyu/CountOfPositivesSumOfNegatives.py
#!/usr/bin/python3

# https://www.codewars.com/kata/576bb71bbbcf0951d5000044/
def count_positives_sum_negatives(arr):
    return [len([x for x in arr if x > 0]), sum(x for x in arr if x < 0)] if arr else []

File: solutions/scala/8 kyu/CountOfPositivesSumOfNegatives.scala
// https://www.codewars.com/kata/576bb71bbbcf0951d5000044/
object Kata {
  def countPositivesSumNegatives(integers: Array[Int]): (Int, Int) = {
    val grouped = integers.groupBy(_>0)
    (if (grouped.contains(true)) grouped(true).size else 0, if (grouped.contains(false)) grouped(false).sum else 0)
  }
}